client/lib/purchases/index.js

```diff
@@ -732,6 +735,38 @@ function getDomainRegistrationAgreementUrl( purchase ) {
 	return purchase.domainRegistrationAgreementUrl;
 }
 
+function shouldRenderMonthlyRenewalOption( purchase ) {
+	if ( ! purchase || ! purchase.expiryDate ) {
+		return false;
+	}
+
+	if ( ! isWpComPlan( purchase.productSlug ) ) {
+		return false;
+	}
+
+	const plan = getPlan( purchase.productSlug );
+
+	if ( TERM_ANNUALLY !== plan.term ) {
+		return false;
+	}
+
+	const isAutorenewalEnabled = ! isExpiring( purchase );
```

Comment:
I checked the backend code for these statuses and it looks like the values can be:
- manual-renew (auto-renew turned off but not expiring)
- expiring (auto renew turned off and expiring)
- auto-renewing (auto-renew turned on)
- active (auto-renew turned on)
- expired

Does it make sense to display the manual renew options to expired subscriptions with auto-renew on?

Reply:
> Does it make sense to display the manual renew options to expired subscriptions with auto-renew on?

IMO, yes, since it is a lower commitment renewal option, it might be attractive to users with expired subscriptions. @paulbonahora, can you please confirm?